hydra-access-controls/lib/hydra-access-controls.rb

```diff
@@ -2,6 +2,7 @@
 require 'active-fedora'
 require 'blacklight'
 require 'cancan'
+require 'action_view' # https://github.com/haml/haml/issues/695
```

Comment:
That would require a new release of Haml. They closed https://github.com/haml/haml/issues/695 several months ago, but no release.